packages/webapp/src/utils/developer-constants.js

```diff
@@ -43,6 +43,11 @@ const ROUTING_REGISTER_OPTIONS = {
   RECORD: 'record-off-site'
 }
 
+const developerEligibilityHTML = {
```

Comment:
@AjinkyaJawalkar 
Is it ok to move this logic in route, instead of defining constant?
cc @z424brave @pwadmore-ea

Reply:
Marking this comment as resolved as per discussed over call